### ISSUE TITLE
Add option on file explorer to open in terminal [new feature request]

### ISSUE THREAD

**Issue by mirageglobe** [adobe/brackets#11229]
In addition to opening in finder....Add in the option of right clicking the file or folder and opening it in terminal. Obviously file will represent opening the parent folder.

This helps in running scripts or opening other file types in a project via command line.

![screen shot 2015-06-08 at 10 21 18](https://cloud.githubusercontent.com/assets/577792/8031660/2df3de7a-0dc8-11e5-8c36-9273e39ea300.png)

**Comment by terraconnect** [comment]
I agree with you. Even more: it would be so valuable to be able to open a terminal window, like e.g. Git bash, right inside Brackets.

**Comment by rbtsolis** [comment]
I agree with you :) +1

**Comment by rbtsolis** [comment]
I find more about this new feature, but if you install Brackets Git Extension: https://github.com/zaggino/brackets-git you
![new brackets feature](https://cloud.githubusercontent.com/assets/8825213/15554213/3c962258-2280-11e6-8422-b0d202d6e0b5.jpg)
 have this feature, see the picture
